seshdash/tasks.py

```diff
@@ -432,3 +431,13 @@ def send_reports():
                              meta = 'REASON FOR FAILURE'
                              )
 
+@shared_task
+def alert_engine():
+    sites = Sesh_Site.objects.all()
+    
+    # TODO check for the latest 10 alerts
+    for site in sites:
+        latest_point = BoM_Data_Point.objects.filter(site_id=site.id).order_by('time')[0]
```

Comment:
getting the latest data point here is not necessary. Line 34 in alert_check you're already creating a object called `latest_instance` this can be for data_point or any other object. Technically there is no difference between Data_point or any other instance.